bootstrap/configs.go

```diff
@@ -69,11 +69,11 @@ type ConfigRepository interface {
 
 	// RetrieveByID retrieves the Config having the provided identifier, that is owned
 	// by the specified user.
-	RetrieveByID(ctx context.Context, owner, id string) (Config, error)
+	RetrieveByID(ctx context.Context, domainID, id string) (Config, error)
 
 	// RetrieveAll retrieves a subset of Configs that are owned
 	// by the specific user, with given filter parameters.
-	RetrieveAll(ctx context.Context, owner string, filter Filter, offset, limit uint64) ConfigsPage
+	RetrieveAll(ctx context.Context, domainID, thingID string, filter Filter, offset, limit uint64) ConfigsPage
```

Comment:
This should be
```suggestion
	RetrieveAll(ctx context.Context, domainID string, thingIDs []string, filter Filter, offset, limit uint64) ConfigsPage
```

Where we can retrieve configs for a specific domain or configs for a list of things.